### DISCUSSION TITLE
Opening Keepass File via URL

### DISCUSSION THREAD

**Discussion by ghost** [keepassxreboot/keepassxc#7206] [Q&A]
**Preinfo:**
So i am Using the Autoopen feature with KeepassXC BUT:
For work context i have several different KDBX files and its a hassle to let them all in the Autoopen as these would be up to 10 different files started from a main file.
Some of the Files are regularly used so they are in the autoopen obviously.
Some files are much less often but i'd like them to be as comfortable as possible.
I am working on a Macbook
I am forced to have all keepassfiles locked automatically within 60 seconds

**The actual challenge:**
can i type something in the URL like:
`file://<path-to-file>/keepassfile.kdbx -pw:{PASSWORD}`
so when i open the file the password set in the entry gets filled automatically?

Sorry in advance if that is a stupid question but it would help my daily work drastically

**Comment by droidmonkey** [comment]
Just do the file part of the url and put the password in the password field of the entry. Double clicking the url will open the database
